policy-test/src/lib.rs

```diff
@@ -17,9 +17,9 @@ pub enum LinkerdInject {
     Disabled,
 }
 
-pub async fn create<T>(client: &kube::Client, obj: T) -> T
+pub async fn create_global<T>(client: &kube::Client, obj: T) -> T
```

Comment:
take it or leave it? it might be nice to have a comment explaining why we have both of these now?